cifar/README.md

```diff
@@ -0,0 +1,177 @@
+# ExecuTorch Model Fine-Tuning on Android App
+
+In this tutorial, we will be fine-tuning a CIFAR 10 model on an android app using ExecuTorch.
+
+## Environment Setup
+
+For the android environment setup, follow these steps:
+
+1. Install Open JDK using: `brew install openjdk@17`
+2. Download and install the latest version of [Android Studio](https://developer.android.com/studio/)  .
+3. Start Android Studio and open the **Settings** dialog (gear icon on the bottom left).
+4. Navigate to **Languages & Frameworks**, then **Android SDK**.
+5. In the **SDK Platforms** tab, check **Android 14.0 (“UpsideDownCake”)** and **Android API 36**.
+6. In the **SDK Tools** tab, check **Android SDK Build-Tools 36**, **NDK (Side by side)**, **Android SDK Command-line Tools (latest)**, **CMake**, **Android Emulator**, and **Android SDK Platform-Tools**.
+7. Export the paths for these to your environment or add them to the `.bashrc` or `.zshrc` files:
+```bash
+# JAVA config
+export JAVA_HOME=/opt/homebrew/opt/openjdk@17/
```

Comment:
This is macOS and version specific, we shouldn't include that. We can reduce these steps to:

```
Ensure the following environment variables are set: `JAVA_HOME`, `ANDROID_NDK`, `ANDROID_SDK`
```

Better yet we can enforce that in our build scripts ;)